src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -144,13 +146,7 @@ export class HttpResourceFetcher implements ResourceFetcher {
 
         promise.finally(() => cancelListener?.dispose())
 
-        return promise.catch((err: RequestError | CancelError) => {
-            // Cancel error has no sensitive info
-            if (err instanceof CancelError) {
-                throw err
-            }
-            throw { code: err.code } // Swallow URL since it may contain sensitive data
```

Comment:
For reference: security review has asked us to avoid logging URLs (e.g. Lambda download URLs have a time-based key in their parameters, IIRC). I guess this error was caught and logged somewhere (but that's a practice we want to avoid anyway...)